### DISCUSSION TITLE
Welcome to whatsapp-monitor Discussions!

### DISCUSSION THREAD

**Discussion by rizwansoaib** [rizwansoaib/whatsapp-monitor#59] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Feature Request 

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Whateverling** [comment]
Hello, probably a stupid question, but I could not figure it out on my own. How do I choose the contacts I want to monitor? It seems that it will pick some contacts at random, like it will just start monitoring every chat I'm in where someone comes online, including groups. What I would like is to figure out how to monitor a single contact of my choosing. 

Btw, what is the difference between the red and the green heart icon?

Using chrome extension on w10.

Thank you.

**Reply by mayursdev** [reply]
i really appreciate your work Rizwan. i just installed desktop version and was wondering how to enable web push notifications in desktop latest version. in earlier version i think it was there to enter the channel, now it isn't. thank you again

**Comment by rizwansoaib** [comment]
Sorry
Desktop App don't support web push notifications.

On Sun 13 Dec, 2020, 3:27 AM webdev111, <notifications@github.com> wrote:

> i really appreciate your work Rizwan. i just installed desktop version and
> was wondering how to enable web push notifications in desktop latest
> version. in earlier version i think it was there to enter the channel, now
> it isn't. thank you again
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/59#discussioncomment-200887>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC2EZGSDK2NMWKQ36BOWCLSUPROLANCNFSM4UUKRQIQ>
> .
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.

**Comment by Naobaz** [comment]
Hi Bro, your new version 2.4 isn't working at all after update........the previous version 2.3 was working fine but as I updated it to 2.4 it's not showing whose online and the time stamps in the monitor mode which comes when we press "Start Monitor". How do I make it work?? Thanks!

**Comment by rizwansoaib** [comment]
I have checked personally and test


Please send screenshot which version you are using


v2.4 working fine please check again

Thanks & regards
RIZWAN AHMAD

On Fri 26 Feb, 2021, 10:46 AM Naobaz, <notifications@github.com> wrote:

> Hi Bro, your new version 2.4 isn't working at all after update........the
> previous version 2.3 was working fine but as I updated it to 2.4 it's not
> showing whose online and the time stamps in the monitor mode which comes
> when we press "Start Monitor". How do I make it work?? Thanks!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/59#discussioncomment-408066>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC2EZC3UMPX5WTWI6N53RLTA4VCXANCNFSM4UUKRQIQ>
> .
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.

**Reply by Naobaz** [reply]
I have uninstalled and reinstalled the .exe file again and tried but nothing is recorded in the online monitor mode(black screen one). For the extension I did the "load unpacked" files from the version 2.4 WhatsApp-Monitor but not showing any notification.

**Comment by rizwansoaib** [comment]
Installed exe is old not new because in new version written V1.2 on tracker
window and Notification

Thanks & regards
RIZWAN AHMAD

On Fri 26 Feb, 2021, 6:19 PM Naobaz, <notifications@github.com> wrote:

> I have uninstalled and reinstalled the .exe file again and tried but
> nothing is recorded in the online monitor mode(black screen one). For the
> extension I did the "load unpacked" files from the version 2.4
> WhatsApp-Monitor but not showing any notification.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/59#discussioncomment-409115>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC2EZFOKOYSTW34GGO5HBTTA6KEPANCNFSM4UUKRQIQ>
> .
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.

**Reply by Naobaz** [reply]
Thank you so much!!! U R D Best!!!!

**Comment by rizwansoaib** [comment]
Install latest version from chrome store
...



No one send me mail regarding not working...

Please try to install from store.

Thanks & regards
RIZWAN AHMAD

On Fri 26 Feb, 2021, 7:04 PM Rizwan Ahmad, <rizwan.178208@knit.ac.in> wrote:

> Installed exe is old not new because in new version written V1.2 on
> tracker window and Notification
>
> Thanks & regards
> RIZWAN AHMAD
>
> On Fri 26 Feb, 2021, 6:19 PM Naobaz, <notifications@github.com> wrote:
>
>> I have uninstalled and reinstalled the .exe file again and tried but
>> nothing is recorded in the online monitor mode(black screen one). For the
>> extension I did the "load unpacked" files from the version 2.4
>> WhatsApp-Monitor but not showing any notification.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/59#discussioncomment-409115>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AHC2EZFOKOYSTW34GGO5HBTTA6KEPANCNFSM4UUKRQIQ>
>> .
>>
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.

**Comment by pleistoneun** [comment]
After starting, I get the error "TypeError: Cannot read property 'Presence' of undefined" in the console. I think, the problem is in the "window.Store" - but the websocket.js loads correctly. Can you help me please?

**Comment by rizwansoaib** [comment]
Please update with new version

Thanks & regards
RIZWAN AHMAD

On Sat 27 Feb, 2021, 12:55 PM pleistoneun, <notifications@github.com> wrote:

> After starting, I get the error "TypeError: Cannot read property
> 'Presence' of undefined" in the console. I think, the problem is in the
> "window.Store" - but the websocket.js loads correctly. Can you help me
> please?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/59#discussioncomment-411783>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC2EZA4GX47Q52IT7D2YQDTBCM5VANCNFSM4UUKRQIQ>
> .
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.

**Reply by pleistoneun** [reply]
Problem solved, i used code-elements from your new version v2.4 - changes in websocket.js and protobuf.js were relevant. Thank you!

**Comment by rizwansoaib** [comment]
You check latest windows exe first uninstall previous version then install
latest version v1.2 exe from repo

Thanks & regards
RIZWAN AHMAD

On Sat 27 Feb, 2021, 5:39 PM pleistoneun, <notifications@github.com> wrote:

> Thank you for your answer. I have installed the version v2.4 and i use
> chrome for windows. The error persists. Do you have another recommendation?
> Best regards.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/59#discussioncomment-412227>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC2EZFYHL45QJJGYZI3JVLTBDOI3ANCNFSM4UUKRQIQ>
> .
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.

**Reply by maxmuster001** [reply]
whatsappanalysis.in  is offline?

**Comment by rizwansoaib** [comment]
Sorry I have no money to scale server databases has 7Million entries and
database had crashed.

Thanks & regards
RIZWAN AHMAD

On Thu 1 Apr, 2021, 2:48 PM maxmuster001, ***@***.***> wrote:

> whatsappanalysis.in is offline?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/59#discussioncomment-556677>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC2EZDPDIUJXI6ZEP5LEJLTGQ247ANCNFSM4UUKRQIQ>
> .
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.

**Reply by maxmuster001** [reply]
Thank you for your commitment

**Reply by rizwansoaib** [reply]
> Thank you for your commitment

We have moved Database with AWS Free tier RDS MySQL . I hope website will not going down .

**Reply by maxmuster001** [reply]
Thanks

**Reply by maxmuster001** [reply]
whatsappanalysis.in is often offline again.

**Comment by rizwansoaib** [comment]
It's always run on AWS Free tier in 3-4months my account got suspended .

I have financial issue I am not able to support it.



Thanks & regards
RIZWAN AHMAD

On Sun 2 May, 2021, 1:45 PM maxmuster001, ***@***.***> wrote:

> whatsappanalysis.in is often offline again.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/59#discussioncomment-684400>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC2EZCGDCZJQWYMPVCIRBTTLUC2RANCNFSM4UUKRQIQ>
> .
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.

**Comment by rizwansoaib** [comment]
Website is Active now Thanks for report.


🙏


On Sun 2 May, 2021, 1:45 PM maxmuster001, ***@***.***> wrote:

> whatsappanalysis.in is often offline again.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/59#discussioncomment-684400>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC2EZCGDCZJQWYMPVCIRBTTLUC2RANCNFSM4UUKRQIQ>
> .
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.

**Comment by RohanSingh1234abcdef** [comment]
Can this be used if a contact has blocked me and I want to monitor that contact?

**Comment by rizwansoaib** [comment]
Nope
Change whatsapp web numbers for that

Thanks & regards
RIZWAN AHMAD

On Wed 12 May, 2021, 2:30 PM RohanSingh1234abcdef, ***@***.***>
wrote:

> Can this be used if a contact has blocked me and I want to monitor that
> contact?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/59#discussioncomment-727822>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC2EZFLZFX2OFL64NHN5QTTNI7TBANCNFSM4UUKRQIQ>
> .
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.

**Reply by RohanSingh1234abcdef** [reply]
How do I change whatsapp web numbers? I have only one phone number with which I am registered on whatsapp.. any other way you know of?

**Comment by RohanSingh1234abcdef** [comment]
Will I appear "Online" to other people while I am monitoring someone else?

**Reply by rizwansoaib** [reply]
no

**Comment by pleistoneun** [comment]
Hello, really nice work.
I installed the latest version 2.6 and got the error "TypeError: Cannot read property 'Presence' of undefined".
I think, its a problem with the "websocket.js", "window.Store".
Please help to correct the error.
THANK YOU!

**Reply by pleistoneun** [reply]
Problem solved! Thanks!
